include/ScanPoint.hpp

```diff
@@ -11,23 +11,28 @@
 class ScanPoint {
 public:
   ScanPoint(); // default constructor
-  void set_xyz( double x, double y, double z ){ fX=x; fY=y; fZ=z; }
+  void set_xyz( double x, double y, double z,double time_1,double t_ext2 ){ fX=x; fY=y; fZ=z;fT_ext2=t_ext2;
```

Comment:
Non-accelerometer related? Should include such changes in a different PR, or include in the title/description of this one.

Reply:
Definately accelerometer related, it is how we get the values of the accelerometer in the ptf_analysis file